modules/swagger-codegen/src/main/resources/rust/api.mustache

```diff
@@ -0,0 +1,102 @@
+{{>partial_header}}
+use std::rc::Rc;
+use std::borrow::Borrow;
+
+use hyper;
+use serde_json;
+use futures;
+use futures::{Future, Stream};
+
+use super::{Error, configuration, models};
+
+pub struct {{{classname}}}Impl<C: hyper::client::Connect> {
+    configuration: Rc<configuration::Configuration<C>>,
+}
+
+impl<C: hyper::client::Connect> {{{classname}}}Impl<C> {
+    pub fn new(configuration: Rc<configuration::Configuration<C>>) -> {{{classname}}}Impl<C> {
+        {{{classname}}}Impl {
+            configuration: configuration,
+        }
+    }
+}
+
+{{#operations}}
+{{#operation}}
+pub trait {{classname}} {
+    fn {{{operationId}}}(&self, {{#allParams}}{{paramName}}: {{{dataType}}}{{#hasMore}}, {{/hasMore}}{{/allParams}}) -> Box<Future<Item = (), Error = Error>>;
+}
+{{/operation}}
+{{/operations}}
+
+
+{{#operations}}
+{{#operation}}
+impl<C: hyper::client::Connect>{{classname}} for {{classname}}Impl<C> {
+    fn {{{operationId}}}(&self, {{#allParams}}{{paramName}}: {{{datatype}}}{{/allParams}}) -> Box<Future<Item = (), Error = Error>> {
+        let configuration: &configuration::Configuration<C> = self.configuration.borrow();
+        let mut req = hyper::Request::new(
+            hyper::Method::{{httpMethod}},
+            format!("{}{{path}}", configuration.base_path).parse().unwrap());
```

Comment:
Yes. It would look like `format!("{}{}", configuration.base_path, path)` assuming that `path` was defined in the function argument.